.github/workflows/functional-all.yml

```diff
@@ -57,6 +57,8 @@ jobs:
     - name: Install devstack
       run: |
         cd devstack
+        # Pin a specific version until we fix it.
+        git checkout 46fd6448978897c9e215fc5e038b3d91daa8db5a
```

Comment:
Ack let's do this for now, but i don't think Devstack considers itself broken so this issue will likely persist for us, i'll report a tech debt issue to remove the pin.